src/TelegramLog.php

```diff
@@ -10,6 +10,7 @@
 
 use Monolog\Logger;
 use Monolog\Handler\StreamHandler;
+use Longman\TelegramBot\Exception\TelegramLogException;
 
 namespace Longman\TelegramBot;
```

Comment:
`namespace` declaration must be before the `use` ones.
Otherwise the `TelegramLogException` can't be found, because it's searched for as `Longman\TelegramBot\TelegramLogException` (missing the `\Exception`)